### PR TITLE
feat(admin): new dashboard design

### PR DESCRIPTION
**Pull request by mxstbr**
Current status:

Screenshot 2020-07-30 at 11 53 13

(Note: I have no clue what's going on with that "A" in "Gatsby Admin", it's some kind of funky Inter font bug)

### CODE REVIEW

File: packages/gatsby-admin/src/components/providers.tsx [hunk 1]
```diff
@@ -7,15 +7,26 @@ import { createUrqlClient } from "../urql-client"
 
 const baseTheme = getTheme()
 
+console.log(baseTheme)
```
Comment (ascorbic) [inline packages/gatsby-admin/src/components/providers.tsx:10]: Log!

File: packages/gatsby-admin/src/pages/index.tsx [hunk 2]
```diff
@@ -1,19 +1,30 @@
...
   DropdownMenu,
   DropdownMenuButton,
   DropdownMenuItem,
   DropdownMenuItems,
+  HeadingProps,
+  Text,
+  Badge,
+  Spacer,
 } from "gatsby-interface"
+// TODO: teach TypeScript about SVGs
```
Comment (ascorbic) [inline packages/gatsby-admin/src/pages/index.tsx:18]: This is as simple as adding a d.ts file somewhere in the root with `declare module "*.svg"`

**Comment by ascorbic** [comment]
Love this. Not needed now, but be aware that at @pvdz 's urigin, we're going to be banning arrow function components at some point soon, to make it easier to profile code. They'll need to change to `export function MyComponent(props:React.PropsWithChildren<Props >):JSX.Element {...}`. There will be a codemod though to fix it.

**Comment by mxstbr** [comment]
> Accessibility review is showing me the color contrast for the grey font on top of the chosen background color is not enough though. Can we fix that?

Yep I saw the same thing and meant to left a comment in Figma for Nachi about it but forgot! Done that now 👍

**Pull request merged**